entity_api/src/query.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{error::Error, QueryFilterMap};
+use sea_orm::strum::IntoEnumIterator;
+use sea_orm::{ColumnTrait, DatabaseConnection, EntityTrait, QueryFilter};
+
+/// Find all records of an entity by the given query filter map.
+pub async fn find_by<E, C>(
```

Comment:
This is some really clean, readable code, and it's in the style of how SeaORM approaches working with entities as well.